### DISCUSSION TITLE
Improve docs for atomWithStorage

### DISCUSSION THREAD

**Discussion by wherehows** [pmndrs/jotai#1749] [General] [answered]
What do you think about adding information about using session storage in a server-side rendering environment in the docs? Personally, I only learned about how to use session storage in a server-side rendering environment after analyzing the jotai internal code.

I solved the problem as follows:

```
const sessionStorage = createJSONStorage(() =>
  typeof window !== 'undefined'
    ? window.sessionStorage
    : (undefined as unknown as Storage)
)
```

Is it too basic of a topic to include?

**Comment by dai-shi** [comment, marked as answer]
Sounds good! Feel free to open a PR.
